README.md

```diff
@@ -8,17 +8,15 @@ Works on most Debian and CentOS/RHEL based distributions!
 1.  Install Raspbian
 2.  Run the command below (downloads [this script](https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh) in case you want to read over it first!)
 
-## `curl -L https://install.pi-hole.net | bash`
+### `curl -sSL https://install.pi-hole.net | bash`
 
 ### Alternative Semi-Automated install
 
 ```bash
 wget -O basic-install.sh https://install.pi-hole.net
-chmod +x basic-install.sh
-./basic-install.sh
+cat basic-install.sh | bash
```

Comment:
Sounds good, just replicating the script that internally calls a `curl` to pipe bash if the script is run by non-root user.